Client/video-nodejs_realtime-chat-client/src/MessageForm.jsx

```
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import { messageService } from './services/messageService.ts';
+import { useAuth } from './components/AuthProvider.tsx';
+
+
+
+
+export const MessageForm = ({  roomId}) => {
+  const [text, setText] = useState('');
+const {  currentUser } = useAuth();
+  return (
+    <form
+      className="field is-horizontal"
+      onSubmit={async (event) => {
+        event.preventDefault();
+        
+      await messageService.sendMessage(text,roomId,currentUser.id)
```

It's good practice to handle potential errors from asynchronous operations. Consider wrapping this `await` call in a `try...catch` block to manage cases where the message fails to send, for example, due to a network issue. This would allow you to provide feedback to the user instead of the application failing silently.